plugins/webp-uploads/tests/test-load.php

```diff
@@ -1087,4 +1087,38 @@ public function test_it_should_generate_fallback_images_for_all_sizes_when_gener
 
 		wp_delete_attachment( $attachment_id );
 	}
+
+	/**
+	 * Create the original mime type for WebP images.
+	 */
+	public function test_that_it_should_convert_webp_to_avif_on_upload(): void {
+		$mime_type = 'image/avif';
```

Comment:
$mime_type doesn't need to be a variable

Reply:
Remove in https://github.com/WordPress/performance/pull/1724/commits/2f65d47b63ecd2d2a4f0cf7197208e99395cdfb3